build/windows/omsupply_server.suf

```diff
@@ -523,7 +669,7 @@ end
 
 -- advance to the next screen
 Screen.Jump(sNextScreenName);
-</Script>
+					</Script>

```

Comment:
don't know why setup factory has reformatted the file. I think it's a new version of sf on the server, so possibly that's why 🤷